recipe/meta.yaml

```
@@ -1,92 +1,104 @@
 {% set name = "langsmith" %}
-{% set version = "0.3.39" %}
+{% set version = "0.4.32" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/langchain-ai/{{ name }}-sdk/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 5095256a32428652183b32d26baa79f13d79f46df7dff9afde52da8819e3983c
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a90bb8297fe0d3c63d9868ea58fe46c52d7e2d1f06b614e43c6a78c948275f24
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install ./python --no-deps --no-build-isolation -vv
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vvv
   skip: True  # [py<39]
 
 requirements:
   host:
     - python
-    - poetry-core
+    - hatchling
     - pip
   run:
     - python
-    - pydantic >=1.0.0,<3.0.0  # [py<312]
-    - pydantic >=2.7.4,<3.0.0  # [py>=312]
-    - requests >=2.0.0,<3.0.0
-    - orjson >=3.9.14,<4.0.0
+    - pydantic >=1,<3
+    - requests >=2.0.0
+    - orjson >=3.9.14  # [python_impl !='pypy']
     - httpx >=0.23.0,<1
-    - requests-toolbelt >=1.0.0,<2
-    - zstandard >=0.23.0,<0.24.0
+    - requests-toolbelt >=1.0.0
+    - zstandard >=0.23.0
     - packaging >=23.2
   run_constrained:
-    - langsmith-pyo3 >=0.1.0,<0.2.0
-    - rich >=13.9.4,<14.0.0
-    - pytest >=7.0.0,<8.0.0
-    - openai-agents >=0.0.3,<0.1
-    - opentelemetry-sdk >=1.30.0,<2.0.0
-    - opentelemetry-api >=1.30.0,<2.0.0
-    - opentelemetry-exporter-otlp-proto-http >=1.30.0,<2.0.0
+    - langsmith-pyo3 >=0.1.0
```

https://github.com/langchain-ai/langsmith-sdk/blob/v0.4.32/python/pyproject.toml#L43
```suggestion
    - langsmith-pyo3 >=0.1.0rc2
```